packages/flutter_tools/lib/src/ios/ios_deploy.dart

```diff
@@ -376,6 +393,7 @@ class IOSDeployDebugger {
           _logger.printTrace(line);
           final bool attachSuccess = line == 'success';
           _debuggerState = attachSuccess ? _IOSDeployDebuggerState.attached : _IOSDeployDebuggerState.detached;
+          _logger.printTrace('Debugger state set to ${attachSuccess ? 'attached' : 'detached'}.');
```

Comment:
Are these logs also print out to real developers? do we plan to remove these new logs after we fix this flake? If so may wanna add a `TODO` to remove them.